Taarafo.Core/Services/Foundations/Profiles/IProfileService.cs

```diff
@@ -15,5 +15,6 @@ public partial interface IProfileService
         ValueTask<Profile> AddProfileAsync(Profile profile);
         IQueryable<Profile> RetrieveAllProfiles();
         ValueTask<Profile> RetrieveProfileByIdAsync(Guid profileId);
+        ValueTask<Profile> ModifyProfileAsync(Profile profile);
```

Comment:
Order the CRUD opetation

Reply:
done I switched the RetrieveAll with the RetrieveById